DecipherNow/helm-charts
# Sense chart cleanup

Changes

- Use a different proxy kubernetes label name. Instead of app, we're using “gm.proxy”
- Update proxy port to use 10808
- Use snake case
- Add comments to all values files
- Expose all envvars in values files, remove all possible envvars from template files
- Clean up all global values

To test:
1. Install secrets
```
helm install secrets secrets --set=global.environment=kubernetes
```
2. Install each chart by itself. Make sure there are no errors on install. Catalog won't come up fully because it's waiting for control
```
helm install sense sense/catalog --set=global.environment=kubernetes
helm uninstall sense
helm install sense sense/dashboard --set=global.environment=kubernetes
helm uninstall sense
helm install sense sense/slo --set=global.environment=kubernetes
helm uninstall sense
```
3. Install the sense umbrella chart. Make sure there are no errors on install and that all pods come up.
```
helm dep up sense
helm install sense sense --set=global.environment=kubernetes
```

```
$ kubectl get pods
NAME                        READY   STATUS     RESTARTS   AGE
catalog-7557dc98b7-sr4q4    0/2     Init:0/1   0          36s
dashboard-779d74844-d92pm   2/2     Running    0          36s
postgres-slo-0              1/1     Running    0          36s
prometheus-0                2/2     Running    0          36s
slo-6d686854d6-jf8qv        2/2     Running    0          36s
```
